runtime/java.go

```diff
@@ -52,23 +51,16 @@ func (conf javaRuntime) Validate() error {
 	return conf.CommonRuntime.Validate(inherit)
 }
 func (conf javaRuntime) GetBootCmd(cmdConfs map[string]*CmdConfig, env map[string]string) (string, error) {
-	cmd := fmt.Sprintf("java.so %s io.osv.isolated.MultiJarLoader -mains /etc/javamains", conf.GetJvmArgs())
-	return conf.CommonRuntime.BuildBootCmd(cmd, cmdConfs, env)
-}
-func (conf javaRuntime) OnCollect(targetPath string) error {
-	// Check if /etc folder is already available. This is where we are going to store
-	// Java launch definition.
-	etcDir := filepath.Join(targetPath, "etc")
-	if _, err := os.Stat(etcDir); os.IsNotExist(err) {
-		os.MkdirAll(etcDir, 0777)
-	}
-
-	err := ioutil.WriteFile(filepath.Join(etcDir, "javamains"), []byte(conf.GetCommandLine()), 0644)
-	if err != nil {
-		return err
-	}
-
-	return nil
+	conf.Base = "openjdk8-zulu-compact1:java"
```

Comment:
Just noticed that this brings a problem with `GetDependencies()` function - if user specifies `base: "openjdk7:java"` then we should make sure that `openjdk7` package is prepended to required list, not `opejdk8-zulu-compact1`. Let me prepare a separate PR with this to solve the problem in general i.e. to prepend referred package automatically when referenced by "base:" attribute.